src/docs/markdown/caddyfile/directives/file_server.md

```diff
@@ -33,7 +33,9 @@ file_server [<matcher>] [browse] {
 	root          <path>
```

Comment:
Should we somehow mention the `embedded` filesystem?

Reply:
Probably yeah. Do you want to write a blurb about it?

Reply:
Will do. I'll include it when I visit this branch for my parts later.